pkg/reconciler/pipelinerun/controller.go

```diff
@@ -13,14 +13,13 @@ func SetupNewReconcilerWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&pipelinev1beta1.PipelineRun{}, builder.WithPredicates(predicate.Funcs{
 			CreateFunc: func(e event.CreateEvent) bool {
-				return true
+				return false
```

Comment:
fwiw I had meant to put TODO commit here as well saying this might be switched to false